src/home_work2/db.js

```diff
@@ -0,0 +1,67 @@
+const db = [];
+
+function getUserById(req){
```

Comment:
As far as I can see, you're not utilising JS context ("this" keyword) so there's nothing stopping you from using arrow functions everywhere.